### DISCUSSION TITLE
[Bug]: Cluster Operator Stuck on Reconciliation of KafkaConnect

### DISCUSSION THREAD

**Discussion by ksa75** [strimzi/strimzi-kafka-operator#10992] [General]
### Bug Description

Cluster Operator Stuck on Reconciliation of KafkaConnect

KafkaConnect pods is not get created after manual deletion
Strimzipodsets exist but the state is empty and does not change
The log of the cluster operator says "reconciliation in progress" and after a while "timeout waiting for Strimzipodset", not much else.
If the cluster operator pod is restarted, kafkaconnect pods are started with significant delay 5min to hour.

For case if size matters:  in this troubled cluster we have ~10 KafkaConnects

### Strimzi version

0.39.0

### Kubernetes version

Kubernetes 1.30.3

### Installation method

helm

### Infrastructure

bare-metal

**Comment by scholzj** [comment]
Please don't open bugs where you don't provide any information - no configs, no logs, nothing.
